### ISSUE TITLE
extends is not defined

### ISSUE THREAD

**Issue by liucui0035** [akveo/ng2-smart-table#26]
When I am using ng2-smart-table, this error occurred. 

I don't konw why the build out files not contains `extend`'s definition.

I installed by npm, `--version 0.2.3`

![image](https://cloud.githubusercontent.com/assets/14086975/18578880/58614b00-7c2d-11e6-8e7e-7156468a7a40.png)

**Comment by Tadwork** [comment]
I am pretty sure that this is because the component is compiled with 
```
    "noEmitHelpers": true,
```
in the tsconfig

 a temporary fix would be to add 
```
  <script>
    var __extends = (this && this.__extends) || function (d, b) {
        for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
        function __() { this.constructor = d; }
        d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
    };
    var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
        var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
        if (typeof Reflect === "object" && typeof Reflect.decorate === "function") r = Reflect.decorate(decorators, target, key, desc);
        else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
        return c > 3 && r && Object.defineProperty(target, key, r), r;
    };
    var __metadata = (this && this.__metadata) || function (k, v) {
        if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);
    };
    var __param = (this && this.__param) || function (paramIndex, decorator) {
        return function (target, key) { decorator(target, key, paramIndex); }
    };
    var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
        return new (P || (P = Promise))(function (resolve, reject) {
            function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
            function rejected(value) { try { step(generator.throw(value)); } catch (e) { reject(e); } }
            function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
            step((generator = generator.apply(thisArg, _arguments)).next());
        });
    };
    </script>

```
as a global script until a new version is pushed to npm

**Comment by bvlangen** [comment]
Adding @Tadwork 's script as a global script (just added it to the index.html head section) did the trick! (For now, as a work-around) Thanks for that! Now I can continue experimenting with the ng2-smart-table.

**Comment by liucui0035** [comment]
@Tadwork Thank you very much.
I have resolved by `import ('ts-helper')`  after I reading the source code of their demo page.

I think it is better to add this tip in the `Read me`

**Comment by nnixaa** [comment]
Thanks @Tadwork, we will update the library accordingly!

**Comment by sambanthar** [comment]
the script is working thanks

**Comment by nnixaa** [comment]
fixed in 0.4.0-3
